### PR TITLE
Enhance year display when min and max are used

### PR DESCRIPTION
**Pull request by sdussaut**
Proposed change for #48

### CODE REVIEW

**Comment by johnantoni** [comment]
@WickyNilliams Tested and working well. Can this be merged into master?

Screen Shot 2020-11-19 at 3 00 08 PM

**Comment by johnantoni** [comment]
@viljamis Any chance this can be merged in? really need this

File: src/components/duet-date-picker/duet-date-picker.tsx [hunk 1]
```diff
@@ -518,13 +518,18 @@ export class DuetDatePicker implements ComponentInterface {
...
 
-    let minYear = selectedYear - 10
-    let maxYear = selectedYear + 10
+    let minYear = selectedYear
+    let maxYear = selectedYear
     if (minDate) {
-      minYear = Math.max(minYear, minDate.getFullYear())
+      minYear = Math.min(minYear, minDate.getFullYear())
+    } else {
+      minYear -= 10
```
Comment (WickyNilliams) [inline src/components/duet-date-picker/duet-date-picker.tsx:526]: If I'm reading it right, can we simplify this logic:

```js
const minYear = minDate ? minDate.getFullYear() : selectedYear - 10
const maxYear = maxDate ? maxDate.getFullYear() : selectedYear + 10
```

My thinking is if there is a min/max date set, we should always use that and ignore the selected year.

Comment (sdussaut) [reply src/components/duet-date-picker/duet-date-picker.tsx:526]: Sounds good to me.

**Comment by johnantoni** [comment]
can this be merged in @WickyNilliams

**Comment by uhrohraggy** [comment]
> can this be merged in @WickyNilliams

+1 am currently using `sdussaut:master` for this exact use case. but would love to use the official version

**Comment by johnantoni** [comment]
same here, don't want to pressure anyone but would really love to get this one in. solves so many problems on my end.

**Comment by johnantoni** [comment]
hi @viljamis can this get merged in? looks ready to roll

**Comment by WickyNilliams** [comment]
Excuse the delay! I'll merge and release this today. Just tested and it works great. Thanks!

**Comment by WickyNilliams** [comment]
Published as `v1.2.0`

**Pull request merged**